src/api/main.py

```diff
@@ -45,12 +55,11 @@ def run_rwd(lat, lon):
     num_processors = '1'
     data_path = '/opt/rwd-data'
```

Comment:
Let's update this to be `/opt/rwd-data/drb` so that it's a sibling with `/opt/rwd-data/nhd` for the sake of being explicit.